internal/common/baseclient/iigo.go

```diff
@@ -83,12 +83,17 @@ func (c *BaseClient) TaxTaken(shared.Resources) {
 	// Just an update. Ignore
 }
 
-// GetTaxContribution FIXME: Add documentation. What does this function do?
+// GetTaxContribution gives value of how much the island wants to pay in taxes
```

Comment:
can we get a new PR that fixes #160

Reply:
Ok yeah, thanks for bringing it to my attention. If I understand correctly the BASE|OPTIONAL|COMPULSORY note is with regards to whether it needs to be overridden?

Reply:
yep :)

Reply:
I think @preetl has addressed most of these in #217, I've pulled main into this branch now

Reply:
hmm, doesn't show for me here but it's fine--we can add later